webpack.config.js

```diff
@@ -228,6 +229,39 @@ const config = (module.exports = {
   },
 
   plugins: [
+    // strip locales, except supported at
+    // https://www.metabase.com/docs/latest/configuring-metabase/localization
+    new MomentLocalesPlugin({
+      localesToKeep: [
+        "sq",
```

Comment:
Let's add Danish
https://metaboat.slack.com/archives/C864UT5CZ/p1715899143489219?thread_ts=1715897850.322739&cid=C864UT5CZ

Reply:
oh yikes, I don't like having to explicitly list these here. we frequently add new ones 😳  we should import them